src/pages/journal/journal-components/journal-candidate/journal-candidate.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'journal-candidate',
+  templateUrl: 'journal-candidate.html',
+  inputs: ['title', 'firstName', 'lastName','testCategory']
+})
+export class JournalCandidateComponent {
+  @Input()
+  title: string;
+
+  @Input()
+  firstName: string;
```

Comment:
Don't we have a name interface that we can pass into this component?

Reply:
We do - in common/domain. Will swap to use this now.